### ISSUE TITLE
Missing Form Label should not flag on image input with alt text

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#610]
https://github.com/equalizedigital/accessibility-checker/blob/183f257855902a96deb54e1b1d51688bfafed9fa/includes/rules/missing_form_label.php#L15

[See post on rule testing site.](https://a11ycheckrules.wpenginepowered.com/missing-form-label/)

If it's an input with a type of image AND alt text, then I would not expect this to flag.

**Comment by pattonwebz** [comment]
@amberhinds I worked though this in #696 and it should be in the next release.

**Comment by amberhinds** [comment]
Thanks!
